src/scripts/modules/backToTop/backToTop.ts

```diff
@@ -0,0 +1,19 @@
+export function backToTop() {
+	const button: HTMLDivElement = document.createElement( 'div' );
+	button.setAttribute( 'id', 'backtotop' );
+	const scrollToTop = () => {
+		window.scrollTo( 0, 0 );
```

Comment:
Although on html or body we have that css property that makes the scroll smooth (scroll-behavior: smooth;), for completeness we will also add the js version

```suggestion
		window.scrollTo({
			top: 0,
			behavior: "smooth"
		});
```